### ISSUE TITLE
tailwindcss CLI v1.3 now requires Node v10

### ISSUE THREAD

**Issue by bradlc** [tailwindcss/tailwindcss#1604]
Because of the [`fs-extra` bump](https://github.com/tailwindcss/tailwindcss/commit/617b3996097e4d186aa0697c3fbe45c9170989f8) the `tailwindcss` command now requires Node v10, as that module [dropped support for Node versions prior to that](https://github.com/jprichardson/node-fs-extra/pull/751).

The specific error I am seeing (on Node `v8.12.0`) is due to the usage of the [optional catch binding](https://2ality.com/2017/08/optional-catch-binding.html) feature added in Node v10:

```
> tailwindcss build in.css -o out.css

node_modules/fs-extra/lib/mkdirs/make-dir.js:86
      } catch {
              ^

SyntaxError: Unexpected token {
```

I'm guessing this was unintentional since it isn't mentioned in the release notes and it would constitute a breaking change. I propose reverting the `fs-extra` bump to maintain the original Node support until at least the next major Tailwind release.

**Comment by adamwathan** [comment]
What do you think of the opinion here that changing language requirements doesn't constitute a new major release?

https://sebastiandedeyne.com/composer-semver-and-underlying-dependency-changes/

I would love to just drop support for Node 8 since it is end-of-life but I don't want to tag 2.0 because of it either. I also don't want dependabot to constantly be nagging me about outdated dependencies by locking to an older version, like it is currently doing to me with Prettier 😅

**Comment by bradlc** [comment]
I think the key difference there is in this paragraph:

> When you run `composer update` from a `php:^7.2` environment, it won't pull in packages that require `php:^7.4`. Composer will keep requiring `my/package:1.0.0` until `my/package:1.0.1` is compatible with your project. Nothing's broken, you just won't be receiving any new updates.

As far as I'm aware this is not the case with Node. If I'm running Node v8 it'll happily update `tailwindcss` to `v1.3.4`, and the CLI won't work anymore.

In composer-land I can get on board with that idea, but in this case I personally think it's a breaking change for sure, because Node/npm doesn't have the same behaviour.

**Comment by adamwathan** [comment]
Fixed in 1.3.5, thanks Brad!
